src/pages/variantEntity/TabClinical.tsx

```diff
@@ -14,7 +14,17 @@ type OwnProps = {
 };
 
 const TabClinical = ({ variantId }: OwnProps) => {
-  const { loading, data } = useTabClinicalData(variantId);
+  const { loading, data, error } = useTabClinicalData(variantId);
+
+  if (error) {
+    return (
+      <Result
+        status="500"
+        title="Server Error"
+        subTitle="An error has occured and we are not able to load content at this time."
+      />
+    );
```

Comment:
used at 3 places at least it has react the magic number to extract

Reply:
corrected